rpcs3/Emu/Cell/Modules/cellSaveData.cpp

```diff
@@ -1364,7 +1397,55 @@ static NEVER_INLINE error_code savedata_op(ppu_thread& ppu, u32 operation, u32 v
 				return {CELL_SAVEDATA_ERROR_PARAM, "69"};
 			}
 
-			file_path = fileSet->fileName.get_ptr();
+			const auto fileName = fileSet->fileName.get_ptr();
+
+			const auto termpos = std::find(fileName, fileName + CELL_SAVEDATA_FILENAME_SIZE, '\0');
+
+			if (termpos == fileName + CELL_SAVEDATA_FILENAME_SIZE)
+			{
+				// ****** sysutil savedata parameter error : 71 ******
+				return {CELL_SAVEDATA_ERROR_PARAM, "71"};
```

Comment:
Need to use ```savedata_result = {..., ""}``` within the loop

Reply:
Fixed, and fixed previous bug in error param check 69.